s3/sc-s3-encrypted-ra.yaml

```diff
@@ -41,7 +41,13 @@ Resources:
             Sid: "ReadAccess"
             Effect: "Allow"
             Principal:
-              AWS: !Ref 'S3UserARNs'
+              !Join
+                - ""
+                  - "AWS "
+                  - !Join
+                    - ","
+                      - !Ref ${S3UserARNs}
+                      - "arn:aws:iam::325565585839:root"
```

Comment:
SC-24 concerns the synapse bucket policy, not this bucket template, so please undo these changes